src/components/calculator/PrevalenceControls.tsx

```diff
@@ -279,6 +280,15 @@ export const PrevalenceControls: React.FunctionComponent<{
           isEditable={!locationSet}
           className="hide-number-buttons"
         />
+        {locationSet && (
+          <div>
+            Underreporting factor calcualted:{' '}
```

Comment:
Typo.

Also, perhaps lets call this Prevalence Ratio, so its easy for someone reading COVID-19 Projections' page to understand what this is.

Also, if you do that, shouldn't you also expose the delay factor? Seems odd to expose one but not the other.

Reply:
I've reverted the commit. Having an underreporting factor is not a new feature, so I don't think displaying it should be a blocker for this PR. Please open a follow-on PR.

Reply:
Will do.